ui/js/services/perf/compare.js

```diff
@@ -217,62 +217,52 @@ treeherder.factory('PhCompare', [
                 return errors;
             },
 
-            getResultsMap: function (projectName, seriesList, params) {
-                var resultsMap = {};
-                return $q.all(_.chunk(seriesList, 20).map(function (seriesChunk) {
-                    if (params.pushIDs) {
-                        return PhSeries.getSeriesData(
-                            projectName, {
-                                signatures: _.map(seriesChunk, 'signature'),
-                                framework: _.uniq(_.map(seriesChunk, 'frameworkId')),
-                                push_id: params.pushIDs }
-                        ).then(function (seriesData) {
-                            params.pushIDs.forEach(function (resultSetId) {
-                                if (resultsMap[resultSetId] === undefined) {
-                                    resultsMap[resultSetId] = {};
+            getResultsMap: (projectName, seriesList, params) => {
+                let resultsMap = {};
+                return $q.all(_.chunk(seriesList, 20).map(
+                    seriesChunk => PhSeries.getSeriesData(
```

Comment:
Previously, before calling PhSeries.getSeriesData, params.pushIDs was verified to exist, would that cause a problem not checking that first?

Reply:
That check was part of an if/else construction which I managed to eliminate by passing the parameters directly into the function (and then from there into the api request). So pretty sure this is ok. :)